transmart-batch/docs/data_formats/expression.md

```diff
@@ -11,44 +11,36 @@ Annotation
 Parameters file name has to be `mrna_annotation.params`.
 `annotation.params` name is also supported as a legacy.
 
-**Attention:** The data file referred by `ANNOTATIONS_FILE` field  of the `mrna_annotation.params` file is expected to have a header (**first line is skipped**).
+**Note:** The data file referred by `ANNOTATIONS_FILE` field  of the `mrna_annotation.params` file is expected to have a header (**first line is skipped**).
 While the data file referred from `annotation.params` **must not have a header**.
 Such distinction made for backward compatibility reasons: old platform files did not have a header.
 
-The annotation data file has to have following columns:
+The annotation file must contain the following columns:
 
 |   Column Name   |            Description             |
 |-----------------|------------------------------------|
-| GPL_ID          | Platform id (e.g. `GPL570`)        |
-| PROBE_NAME      | Probe name (e.g. `1007_s_at`)      |
-| GENES           | Gene name (e.g. `DDR1`)            |
-| ENTREZ_IDS      | Entrez gene id (e.g. `780`)        |
-| ORGANISM        | Species name (e.g. `Homo Sapiens`) |
-
-
-Data
-----
-
-### Subject Sample Mapping
-
-See [subject-sample-mapping.md](subject-sample-mapping.md)
+| GPL_ID          | Platform ID (e.g. `GPL570`). Has to be identical for all rows.       |
+| PROBE_NAME      | Depending on level of aggregation, either a probe or gene name (e.g. `1007_s_at`). Has to be unique.      |
+| GENE_SYMBOL     | HGNC gene symbol (e.g. `DDR1`).            |
+| ENTREZ_ID       | Entrez gene id (e.g. `780`)        |
+| ORGANISM        | Scientific name of the species the platform is based on (e.g. `Homo sapiens`) |
 
 
 ### Expression Data
 
 This is the data file format:
 
-| `ID_REF`    | `GSM210005` | `GSM210006` | `GSM210007` |
+| ID_REF    | GSM210005 | GSM210006 | GSM210007 |
```

Comment:
It's not like anyone makes these tables by hand I hope ;) 
But alright, I fixed it.